modules/steam.nix

```
@@ -98,5 +109,21 @@ in
         partOf = [ "graphical-session.target" ];
       };
     })
+    (mkIf (config.jovian.steam.enable && config.jovian.steam.enableUdevRules) {
```

```suggestion
    (mkIf (config.jovian.steam.enableUdevRules) {
```

The rules shouldn't require using steam. If an alternative UI wants to control the same toggles and knobs, it should be possible without enabling steam.

---

Sure, let's move all of the Steam Deck-specific configurations into a the `steamdeck` subdirectory, with options under `jovian.devices.steamdeck`. I'll do that in a separate PR.